src/game/mod.rs

```diff
@@ -56,35 +55,8 @@ impl EventHandler for State {
         keycode: KeyCode,
         _keymod: KeyMods
     ) {
-        match keycode {
-            KeyCode::W => {
-                self.player1.movedir(Directions::Up);
-            },
-            KeyCode::A => {
-                self.player1.movedir(Directions::Left);
-            },
-            KeyCode::S => {
-                self.player1.movedir(Directions::Down);
-            },
-            KeyCode::D => {
-                self.player1.movedir(Directions::Right);
-            },
-            _ => (),
-        }
-        match keycode {
-            KeyCode::I => {
-                self.player2.movedir(Directions::Up);
-            },
-            KeyCode::J => {
-                self.player2.movedir(Directions::Left);
-            },
-            KeyCode::K => {
-                self.player2.movedir(Directions::Down);
-            },
-            KeyCode::L => {
-                self.player2.movedir(Directions::Right);
-            },
-            _ => (),
-        }
+        /*
```

Comment:
Pretty much the only part where we could talk a little bit about pattern matching in general/ in Rust and why enums are cool :D

Reply:
Oh yes, I agree! This is definitely something I want explicitely done by attendees because it's cool :D